### PR TITLE
updated groups to work with new benchmarks

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

**Comment by chaudum** [comment]
@m-kharbat or @kovrus can you take a look!

File: pages/benchmark/index.html [hunk 1]
```diff
@@ -1,160 +1,87 @@
...
-    (4 cores 2.4 GHz Intel Xeon&reg; E5-2673 v3 Haswell machine with 28 GB RAM and SSD).</p>
+<p>This page shows the results of the Crate benchmarks based on the master
+  branch (nightly builds) as of that point in time. The benchmarks are executed
+  as <code>cr8</code> benchmarks on Microsoft Azure cloud environment Standard_DS12 instances
+  (4 cores 2.4 GHz Intel Xeon&reg; E5-2673 v3 Haswell machine with 28 GB RAM and SSD).</p>
+
+<p>The benchmark setup is as follows: 1 Crate instance,
+  running with default settings on a single host:
+  <ul>
+    <li><code>CRATE_HEAP_SIZE</code> is set to 4 GB</li>
```
Comment (kovrus) [inline pages/benchmark/index.html]: bump the heap size

File: pages/benchmark/index.html [hunk 2]
```diff
@@ -1,160 +1,87 @@
...
 {% endblock %}
 
 {% block body %}
-    <p>This page shows the results of the Crate.IO benchmarks based on the master
-    branch (nightly builds) as of that point in time. The benchmarks are executed
-    as JUnit benchmarks on Microsoft Azure cloud environment Standard_DS12 instances
-    (4 cores 2.4 GHz Intel Xeon&reg; E5-2673 v3 Haswell machine with 28 GB RAM and SSD).</p>
+<p>This page shows the results of the Crate benchmarks based on the master
+  branch (nightly builds) as of that point in time. The benchmarks are executed
+  as <code>cr8</code> benchmarks on Microsoft Azure cloud environment Standard_DS12 instances
```
Comment (kovrus) [inline pages/benchmark/index.html]: shall we point to the `cr8` tool?

**Comment by kovrus** [comment]
otherwise ack

**Pull request merged**